runtime/compiler/x/amd64/codegen/AMD64JNILinkage.cpp

```diff
@@ -1350,6 +1350,18 @@ TR::Register *J9::X86::AMD64::JNILinkage::buildDirectJNIDispatch(TR::Node *callN
 
       if (isGPUHelper)
          callNode->setSymbolReference(callSymRef); //change back to callSymRef afterwards
+
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
On monitor entry and exit code, we only update the counter if it is >= Java19 and 64-Bit platform, should we be guarding the code with 64-Bit platform check as well?

Reply:
No.  This file is specific to 64-bit only (AMD64) and an additional 64-bit check would be redundant.

Reply:
I should have look at that more carefully, thanks a lot for pointing this out. 
Though, I actually want to see if @gacholio's suggestion on Z PR for Loom https://github.com/eclipse-openj9/openj9/pull/15752#issuecomment-1252540101 also applies to all other codegens?

Reply:
This is really a policy decision, not a technical one. It's quite unlikely that 32-bit will make a return, though Z is perhaps the one place where it might be requested.

Reply:
Thanks a lot @gacholio  for the explanation. It makes sense.